Source/rclUE/Public/ROS2Node.h

```diff
@@ -35,17 +37,30 @@ DECLARE_DYNAMIC_DELEGATE_OneParam(FActionCallback, UROS2GenericAction*, InAction
 DECLARE_DELEGATE(FSimpleCallback);
 
 /**
- * @brief Helper structs which is components of the node and should register to 
+ * @brief RR_ROS2_SUBSCRIBE_TO_TOPIC
+ * FSubscriptionCallback is of dynamic delegate type to be serializable for BP use
+ * FSubscriptionCallback::BindDynamic is a macro, instead of a function.
+ * Thus InCallback can only be a direct UFUNCTION() method & cannot be used as typed param!

```

Comment:
Compiler warns if InCallback is not a direct UFUNCTION ?

Reply:
Yeah, it is quite a cruft that for BP-usable dynamic delegate [binding](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Public/Delegates/Delegate.h#L420) is written in a macro, which manually extracts the literal function name then store it [here](https://github.com/EpicGames/UnrealEngine/blob/release/Engine/Source/Runtime/Core/Public/Delegates/DelegateSignatureImpl.inl#L1038)
-> As such, in order to implement a type-based method, we need to hook up directly to `__Internal_BindDynamic`, which is not recommended & have to pass the literal function name ("Foo") along with the method pointer (`&AClass::Foo`)